cibyl/cli/output.py

```diff
@@ -27,7 +27,7 @@ class OutputStyle(Enum):
     """A machine-readable text based format."""
 
     @staticmethod
-    def from_key(key):
+    def from_key(key: object) -> 'OutputStyle':
```

Comment:
I have already documented the parameter types in the docstring, there is no need to do it again. Having this information twice will only get us inconsistency between the two and more maintenance work.

Reply:
If I remove it I get "cibyl/cli/output.py:30: error: Function is missing a type annotation" from mypy